test_collections/sdk_tests/support/python_testing/sdk_python_test_tests.py

```diff
@@ -0,0 +1,110 @@
+#
+# Copyright (c) 2023 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from pathlib import Path
+
+from loguru import logger
+
+from .models.python_test_folder import PythonTestFolder
+from .models.python_test_parser import PythonParserException, parse_python_test
+from .models.test_declarations import (
+    PythonCaseDeclaration,
+    PythonCollectionDeclaration,
+    PythonSuiteDeclaration,
+)
+from .models.test_suite import SuiteType
+
+###
+# This file hosts logic load and parse Python test-cases, located in
+# `test_collections/sdk_tests/sdk_checkout/python_testing/scripts/sdk`.
+# The `sdk` sub-folder here is automatically maintained using the
+# `test_collections/sdk_tests/fetch_sdk_tests_and_runner.sh` script.
+#
+# The Python Tests are organized into 1 Test Suite:
+#        - Automated
+###
+
+SDK_PYTHON_TEST_PATH = Path(
+    "/app/backend/test_collections/sdk_tests/sdk_checkout/python_testing/scripts/sdk"
+)
```

Comment:
Please use the newly created `SDK_CHECKOUT_PATH` from `test_collections.sdk_tests.support.paths`